base/sysinfo.jl

```diff
@@ -162,7 +162,7 @@ total_memory() = ccall(:uv_get_total_memory, UInt64, ())
 Get the process title. On some systems, will always return empty string. (not exported)
 """
 function get_process_title()
-    buf = zeros(UInt8, 512)
+    buf = Vector{UInt8}(512)
```

Comment:
But `unsafe_string` copies  a string from the address of a C-style NUL-terminated string. Whereas if I use `StringVector` I have to manually resize it ?